cigaradvisor/blocks/article-list/article-list.css

```diff
@@ -1 +1,54 @@
-/** Dummy File for now **/
+@import url('../article-teaser/article-teaser.css');
```

Comment:
Don't load the Teaser Css into this file. Instead use the `loadCSS` function to make sure it's loaded into the page.

Reply:
Done